src/server/game/Entities/Player/Player.cpp

```diff
@@ -18986,17 +18991,28 @@ bool Player::CheckInstanceLoginValid()
     {
         // cannot be in raid instance without a group
         if (!GetGroup())
-            return false;
+            return IsInstanceLoginGameMasterException();
```

Comment:
> if player has gm command you will bypass all other checks

this is the whole point of the PR as described in the PR description, I don't get the point of your comment